### ISSUE TITLE
Where I can download expand framework 14.1?

### ISSUE THREAD

**Issue by kanapolw** [eXpandFramework/eXpand#720]
I am using Devexpress 14.1 on my developed application using Visual Studio 2013.  I would like to know where I can find the latest version of expand framework 14.1 to work with Devexpress 14.1

Best Regards,

Thank you.

**Comment by kanapolw** [comment]
More information,  I used to download expandframework 14.1.6.9  many years ago, and i used some feature of them. but now i tried to study how to use Workflow Module on expand framework, and I found bug when it is trying to start application server.

I know that it may be not the most stable version on 14.1 , I can upgrade from 14.1.6 to other version of 14.1.   So I am trying to find the most stable version of expandframework 14.1 for test.

**Comment by apobekiaris** [comment]
all releases are available in the release page http://releases.expandframework.com/
 
using [XpandPwsh]() you can fast discover all published releases for 14.1

```ps1
Get-XpandRelease -Type eXpand -NameMatch 14.1

Sha                                      Name
---                                      ----
5381a0a3e1a1966c22e359ca5f3c76be7e7403e6 14.1.3.2
172dd6d589f345255c2d9ea706874b394afa0aed 14.1.3.3
86224aa28903b351da62ed48a6dfadd287844a33 14.1.3.4
197cc3033dd0234c95a5400730b4bb49246c9a55 14.1.4.1
d812d721dd8095a73aba60ff6653fa3a7a184e7a 14.1.4.2
83a3a2b6c1c4b6872c7051e9c00ea093febec5b6 14.1.4.3
2abb511113174f9e274c28125cffd54ca389e635 14.1.4.4
90bdf4d8e7666f9b64fc5b9424425c30a6501f83 14.1.4.5
25cd65c79d3e71ee56a8f9ea31eb840cdc78f486 14.1.4.7
d5c973100dbf90ad0b4591c3602ada6049a3d84c 14.1.4.8
edc728eeaaaaab5e39451f70d45f184bfd3002ea 14.1.5.1
fa206ff623e7be26642caeead5bbccff90f29a52 14.1.5.2
8f4728a5a59fea3768b35687b74433a31ddf2b15 14.1.5.3
06f309346b13400af28581a955693f15624c731e 14.1.5.4
bf35eb81726fda9a2cb1c328d04ddaddee3e7f60 14.1.6.3
dbbb2129b55ca3312334842ebcdc1d22f6912156 14.1.6.4
01e84a5f8649fcb5cfd9abe9e2f11f6c6f4250e4 14.1.6.5
e01a44ca353e0c1dc723a8f21bca4089577c3190 14.1.6.7
8ec1d7ada90e954264d5dfba3f07852b346b666d 14.1.6.8
3abbca87e01128306151df524013eec5bd207934 14.1.6.9
a299480b212d263ca7bace5cd20b5199d684a02f 14.1.7.1
f636141aa5afc9c2c7e9b398899fce97a63f30fe 14.1.7.3
a2cf726be48bb03b384fd23f266f9ff997379fe9 14.1.7.4
4cdf39f151fe57c0199170dbfb22136ac98aae18 14.1.7.6
1673c1a8a01417d1d9b894206cb4b2f1467307ae 14.1.7.7
a769d938daf30292e436d40749df172625927dab 14.1.7.8
30c183441d98782e47d588180a637f1dd0e0cd35 14.1.7.9
ff7ecea20a829b4a462303942266fec2ece2112f 14.1.8.1
edf2e382e00172ff9f6d5212194d0f67a3ece486 14.1.8.2
```

prefix the version with the repo uri and u can nivigate directly 

```
https://github.com/eXpandFramework/eXpand/releases/tag/YOURVERSION
```

**Comment by kanapolw** [comment]
Thank you, this save me my time.
